faq/faq.md

```diff
@@ -0,0 +1,92 @@
+
+### IntelliJ IDEA CE
+
+JPA Buddy can only retrieve data from a connection string that is explicitly specified. This is because it uses a JDBC driver to obtain meta information. So, if your tables are in a non-default schema, you need to create a new connection that targets the database and schema. You can find instructions on how to do this [here](/database-connections/database-connections.md#non-default-schema-connection).
+
+If your tables are still not visible even after specifying the schema explicitly, please let us know on <a href="https://discord.com/invite/zUsVXYGsQr" target="blank">Discord</a>. When making your request, please provide the version and type of your IntelliJ IDEA (CE or Ultimate), database provider (PostgreSQL, Oracle, MySQL, etc.) and the version of JPA Buddy.
+
+### IntelliJ IDEA Ultimate
+
+Until JPA Buddy version 2023.2, the same mechanism was used for both IntelliJ IDEA Ultimate and IntelliJ IDEA CE. Therefore, if you are using an older version of JPA Buddy and IntelliJ IDEA Ultimate, you need to explicitly specify the schema in the connection string from which you want to create entities as described [here](/database-connections/database-connections.md#non-default-schema-connection).
+
+Starting from version 2023.2, JPA Buddy seamlessly integrates with IntelliJ IDEA capabilities to read metadata about the database. By default, this integration is enabled. If you see tables in the "Database" panel of IntelliJ IDEA, creating entities should work without any problems. You can check if the integration is enabled by going to _Settings -> Tools -> JPA Buddy -> Reverse Engineering -> Integration with IDEA Ultimate database metamodel_.
+
+![seamless-integration.png](img/seamless-integration.png)
+
+If you still can’t see any tables, please contact us on <a href="https://discord.com/invite/zUsVXYGsQr" target="blank">Discord</a>. When making your request, please provide the version and type of your IntelliJ IDEA (CE or Ultimate), database provider (PostgreSQL, Oracle, MySQL, etc.) and the version of JPA Buddy.
+
+## Does JPA Buddy supports the IntelliJ IDEA EAP?
+
+Yes, but unfortunately, we cannot support EAP at early stages. Normally, we provide support for EAP since its 3-4th release. Otherwise, it becomes too laborious, because JetBrains heavily changes API for earlier EAP releases.
+
+## Why am I unable to find certain features?
+
+Certain features in JPA Buddy are only available when specific libraries are present. For instance, you can create Liquibase changelogs or MapStruct mappers only if you have the [corresponding library in the libraries list](/installation-guide/installation-guide.md#dependencies). So, please make sure you have [the required library](/installation-guide/installation-guide.md#dependencies) in your project.
+
+## Is JPA Buddy available for free?
+
+Since we started the plugin, we established a clear list of values that we should deliver to JPA Buddy users:
```

Comment:
```suggestion
JPA Buddy is designed to help users:
* get started with the Jakarta Persistence API and related libraries
* follow best practices
* speed up development
We believe in providing features that support these goals for free.
```
This section could be shorter and a bit more structured.
I suggest the following:
After that, we can add "Our **free features** include: ... <list of features>"
then "Our **subscription package** offers a variety of features designed to accelerate development and deliver direct business value: ... <list of features> you can check the full list here"

Then say "All features of JPA Buddy remain **free for academic use** (teachers, students, classroom assistance) as well as training courses, coding schools and bootcamps. Here is how to obtain it."

The rest seems okay! Check the revised version on github and pick what you like from there or make it shorter like in the suggestions below